scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,12 +32,27 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
     num_of_tables = get_number_of_partitions(dataset_name, project_id)
 
-    for i in range(1, num_of_tables + 1):
+    # For Foxtrot, we need to start at the new_sample_cutoff and export everything remaining in that group and then export the sample_info table in 4000 sample chunks to match with the other tables (vet and ref_ranges)
+    ## we want to know what table number to start with if we have a new sample cutoff
+    ## and we will want groupings of 4000 samples once we have made it from the new sample cutoff to the next group start
+    ## so we will need one avro file of everything from the int(new_sample_cutoff) to the end of the next table, and then the rest of the tables
+    def superpartition_for(sample_id):
+        if sample_id % 4000 == 0:
+            return sample_id // 4000
+        else:
+            return sample_id // 4000 + 1
+
+    start_table = 1 if not new_sample_cutoff else superpartition_for(new_sample_cutoff)
+    for i in range(start_table, num_of_tables + 1):
         file_name = f"*.{i:03}.avro"
-        id_where_clause = f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000}"
+        if new_sample_cutoff:
+            id_where_clause = f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000} AND sample_id > {new_sample_cutoff}"
```

Comment:
```suggestion
            id_where_clause = f"sample_id <= {i * 4000} AND sample_id > {new_sample_cutoff}"
```

Reply:
ok I am dumb, but I dont understand this? Dont we need to bookend the sample_id within the partition--unless it's  the partition that the cutoff is in and then yes this is overkill, but it seems more readable to keep it consistent with the other clause and tack on the sample_cutoff at the end?

Reply:
Discussed in standup, I was wrong, please disregard... ☹️